src/main/kotlin/Menu.kt

```diff
@@ -0,0 +1,61 @@
+abstract class Menu (
+    private var items: ArrayList<out Creatable>?,
+    private val menuName: String,
+    private val menuType: String
+        ) {
+
+    private var counterMenuItems: Int = 0
+    var menuMap: MutableMap<Int, String> = mutableMapOf()
+
+    private fun fillMenu() {
+        clearMenu()
+        addMenuItem("Создать ${menuType}")
+        if (items != null) {
+            for (item in items!!) {
+                addMenuItem(item.name)
+            }
+        }
```

Comment:
🍏 https://kotlinlang.org/docs/null-safety.html#safe-calls
```suggestion
        items?.forEach { item ->
           addMenuItem(item.name)
        }
```